fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2165,4 +2165,10 @@ public class Config extends ConfigBase {
                     + "The larger the value, the more uniform the distribution of the hash algorithm, "
                     + "but it will increase the memory overhead."})
     public static int virtual_node_number = 2048;
+
+    @ConfField(description = {
+            "控制系统自动触发作业执行记录的持久化行数",
```

Comment:
In all descriptions, make it clear that these parameters control the collection of statistics